src/main/java/org/mdbenefits/app/submission/actions/MoveDataIntoSubflowData.java

```diff
@@ -0,0 +1,50 @@
+package org.mdbenefits.app.submission.actions;
+
+import static formflow.library.inputs.FieldNameMarkers.DYNAMIC_FIELD_MARKER;
+
+import formflow.library.config.submission.Action;
+import formflow.library.data.Submission;
+import lombok.extern.slf4j.Slf4j;
+import java.util.List;
+import java.util.Map;
+import java.util.ArrayList;
+
+/**
+ * This is a parent class for `beforeSaveAction` that will move data from a wildcard field into the household member's entry that
+ * it belongs with.
+ */
+@Slf4j
+public abstract class MoveDataIntoSubflowData implements Action {
```

Comment:
I created this to aid in moving data around.  The intent is that someone can create a child Action from this and indicate the fields they are moving things to/from.

Reply:
This will handle those wildcard fields we want to tuck into household.